runtime/src/iree/base/CMakeLists.txt

```diff
@@ -45,11 +45,21 @@ iree_cc_library(
     core_headers
   HDRS
     "alignment.h"
+    "allocator.h"
```

Comment:
same here - not sure what you're doing, but the core_headers explicitly _don't_ have these files